Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/LocalToGlobalConverterUtils.cs

```diff
@@ -28,102 +29,43 @@ public Base TransformObjects(Base atomicObject, List<Matrix4x4> matrix)
+      foreach (var transform in transforms)
       {
-        return TransformObjWithDisplayValues(newObject, matrix);
+        c.TransformTo(transform, out ITransformable newObj);
+        c = newObj;
       }
-      throw new SpeckleException(
-        $"Display Values of types '{listVals.ToList().FindAll(x => x is not SOG.Mesh).Select(y => y.speckle_type).Distinct().ToList()}' for {newObject.speckle_type} are not supported for local to global coordinate transformation"
-      );
-    }
-    // TODO: here or preferably in converters elegantly!
-    // else if (newObject is SOG.Polyline polyline)
-    // {
-    //
-    // }
-    // else if (newObject is SOG.Line line)
-    // {
-    //
-    // }
-    // else if (newObject is SOG.Point point)
-    // {
-    //
-    // }
-    if (newObject is SOG.Mesh mesh)
-    {
-      return TransformMesh(mesh, matrix);
-    }
-    throw new SpeckleException(
-      $"{newObject.speckle_type} is not supported for local to global coordinate transformation"
-    );
-  }
-
-  private Base TransformObjWithDisplayValues(Base baseObj, List<Matrix4x4> matrix)
-  {
-    Base newObject = Core.Api.Operations.Deserialize(Core.Api.Operations.Serialize(baseObj));
-    List<SOG.Mesh> newDisplayValue = new();
 
-    var displayValue = newObject.TryGetDisplayValue();
-    if (displayValue is null) // will not happen due to the check in "TransformObjects"
-    {
-      throw new SpeckleException($"{newObject.speckle_type} blocks contains no display value");
-    }
-
-    foreach (Base displayVal in displayValue)
-    {
-      if (displayVal is SOG.Mesh displayMesh)
-      {
-        var newMesh = TransformMesh(displayMesh, matrix);
-        newDisplayValue.Add(newMesh);
-      }
-      else // will not happen due to the check in "TransformObjects"
+      if (c is not Base)
       {
-        throw new SpeckleException($"Blocks containing {baseObj.speckle_type} are not supported");
+        throw new SpeckleConversionException(
+          $"Blocks transformation of type {atomicObject.speckle_type} did not return a valid object"
+        );
       }
-    }
 
-    newObject["displayValue"] = newDisplayValue;
-    return newObject;
-  }
+      atomicObject = (Base)c;
 
-  private SOG.Mesh TransformMesh(SOG.Mesh displayMesh, List<Matrix4x4> matrix)
-  {
-    List<List<double>> oldVertices = new();
-    for (int i = 0; i < displayMesh.vertices.Count; i += 3)
-    {
-      List<double> group = new();
-      for (int j = 0; j < 3 && i + j < displayMesh.vertices.Count; j++)
+      // .TransformTo only transfers typed properties, we need to add back the dynamic ones:
+      foreach (var prop in atomicObject.GetMembers(DynamicBaseMemberType.Dynamic))
       {
-        group.Add(displayMesh.vertices[i + j]);
+        if (prop.Value is not Base)
+        {
+          atomicObject[prop.Key] = prop.Value;
+        }
+        else
+        {
+          // TODO: add more granular warnings if needed that the property was skipped for blocks
```

Comment:
Why we prevent to add non base props? what if we have some attributes dictionary? I have it on sketchup, if you do it like this, you won't be able to get attributes that attached to blocks

Reply:
hmm, with Base I imagined a geometry being attached. Because those will not be transformed according to the matrices and might be confusing. 
Is there a specific "attributes" base object for blocks?

Reply:
But still you might loose some info from original object. Why we need this if else statement at all?

Reply:
so when we have more granular reporting (e.g. Success status Warnings) we can notify the user

Reply:
we can do with the ticket! My impression is that allowing to pass Base "as is" to all instances will create more tech debt, as we cannot predict what the users will be passing. (e.g. someone creates a "TopRightCorner" property with a Point object, and it will be passed without transformations.)
https://linear.app/speckle/issue/CNX-153/agis-fill-the-placeholders-for-granular-reporting 
But ping me if you want to quickly align on this